### PR TITLE
Multidrop ordering fix

### PR DESCRIPTION
**Pull request by JoeKlesczewski**
This is to fix the multidrop bug described in #13894.

### CODE REVIEW

File: src/inventory_ui.cpp [hunk 1]
```diff
@@ -1080,6 +1091,7 @@ std::list<std::pair<int, int>> game::multidrop()
     }
 
     std::list<std::pair<int, int>> dropped_pos_and_qty;
+	dropped_pos_and_qty.sort(multidrop_comparison);
```
Comment (Coolthulhu) [inline src/inventory_ui.cpp]: Tabs (should be 4 spaces)

Comment (JoeKlesczewski) [inline src/inventory_ui.cpp]: Fixed. My mistake

**Comment by BevapDin** [comment]
This approach might theoretically work, but:

`std::list::sort` sorts the **current** content of the list. If therefor does effectively nothing if the list is empty.

And the list **is** empty, it has just been created via the default constructor:
```C++
std::list<std::pair<int, int>> dropped_pos_and_qty;
dropped_pos_and_qty.sort(multidrop_comparison);
```

**Comment by JoeKlesczewski** [comment]
Wow. That was... very dumb of me, to say the least. I should have that fixed in a moment.

**Comment by kevingranade** [comment]
Did you test the fix?  I don't think it's going to work, because the code that actually drops the items in activity_item_handling.cpp splits the list into a list of worn items and a list of held items itself, then it proceeds to drop the worn items first (oops).  I think I have a fix that will do what we want shortly.

**Comment by OzoneH3** [comment]
Fixed with #14015

**Pull request closed without merging**